From: Andreas Stieger <astieger@suse.com>
Upstream: no

Do not generate timestamp in docs to prevent rebuilds

Index: editorconfig-core-c-0.12.6/doc/Doxyfile.in
===================================================================
--- editorconfig-core-c-0.12.6.orig/doc/Doxyfile.in
+++ editorconfig-core-c-0.12.6/doc/Doxyfile.in
@@ -1279,7 +1279,7 @@ HTML_COLORSTYLE_GAMMA  = 80
 # The default value is: NO.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_MENUS tag is set to YES then the generated HTML
 # documentation will contain a main index with vertical navigation menus that
